tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -124,6 +124,20 @@ def has_python_version(cls, version):
     except OSError:
       return False
 
+  @classmethod
+  def python_interpreter_path(cls, version):
+    """Returns the interpreter path if the current system has the specified version of python.
+
+    :param version: A python version string, such as 2.7, 3.
+    """
+    try:
+      py_path = subprocess.check_output(['python%s' % version,
+                                         '-c',
+                                         'import sys; print(sys.executable)']).strip()
+      return py_path
```

Comment:
Not sure is `os.path.realpath`'ing this would save gymnastics in the use-site, but an idea.